Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml

```diff
@@ -59,7 +59,7 @@
         android:id="@+id/scanCheckInQrCodeFragment"
         android:name="de.rki.coronawarnapp.ui.presencetracing.attendee.scan.ScanCheckInQrCodeFragment"
         android:label="ScanCheckInQrCodeFragment"
-        tools:layout="@layout/fragment_scan_qr_code" />
+        tools:layout="@layout/fragment_scan_qr_code"></fragment>
```

Comment:
why ?

Reply:
Had added nav args before duplicating the Qr fragment and forgot to delete this.